include/zephyr/usb_c/usbc.h

```diff
@@ -146,20 +146,80 @@ enum usbc_policy_wait_t {
 };
 
 /** @cond INTERNAL_HIDDEN */
+/** \addtogroup sink_callbacks
+ *  @{
+ */
+/**
+ * @brief Callback type used to get the Sink Capabilities
+ *
+ * @param dev dev Runtime device structure
```

Comment:
Please specify the type of device passed in for all these prototypes.
```suggestion
 * @param dev USB-C Connector Instance
```

Reply:
Done